tests/frontend/workspace.py

```diff
@@ -175,20 +187,13 @@ def test_open_bzr_customize(cli, tmpdir, datafiles):
 
 @pytest.mark.datafiles(DATA_DIR)
 def test_open_multi(cli, tmpdir, datafiles):
-
     workspace_object = WorkspaceCreator(cli, tmpdir, datafiles)
     workspaces = workspace_object.open_workspaces(repo_kinds)
 
     for (elname, workspace), kind in zip(workspaces, repo_kinds):
         assert kind in elname
         workspace_lsdir = os.listdir(workspace)
-        if kind == "git":
-            assert ".git" in workspace_lsdir
```

Comment:
Do we lose the check that a workspace on a git source has a `.git` directory? Or is such a check irrelevant or redundant (it might make sense to check this in a plugin-specific test)?

Reply:
For git, we have https://github.com/apache/buildstream/blob/master/tests/sources/git.py#L64-L86 that ensures we can successfully run `git`.

For `bzr`, we have https://github.com/apache/buildstream/pull/1565/files that also tangentially ensures we have a .bzr directory.

If you think it's better to have explicit tests, I am happy to add some in the source-specific entries.

Reply:
Tests that run `git`/`bzr` on the workspace directory should indeed be sufficient.